src/hooks/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as useInterval } from './useInterval';
+export { default as useWindowSize } from './useWindowSize';
```

Comment:
Not for this PR, but is `useWindowSize` the same one that Paragon has? Should we remove `useWindowSize` here in favor of the one exported by Paragon now?

Reply:
It is, we should. The paragon one has tests ;)

Reply:
https://openedx.atlassian.net/browse/ENT-4048